src/App.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import './App.scss';
+import { Person } from './components/Person/Person';
```

Comment:
Ensure that the `Person` component can handle cases where some properties might be missing, such as `age` for `olya` and `alex`. Consider providing default values or handling undefined properties within the `Person` component.